tools/cli/go-whisk-cli/commands/activation.go

```diff
@@ -31,12 +31,9 @@ import (
     "github.com/spf13/cobra"
 )
 
-const (
-    PollInterval = time.Second * 2
-    Delay        = time.Second * 5
-)
+const MAX_ACTIVATION_LIMIT = 200
```

Comment:
It looks like merged pr #1980 has the basic structure to include this limit now..  Is there another issue to pull this value from that backend interface?